api/pkg/guard/guard.go

```diff
@@ -1,56 +1,133 @@
-// Package guard is a helper package to evaluate question about members in ShellHub.
+// Package guard is a helper to work around permissions on ShellHub API.
 package guard
 
 import (
-	"errors"
-
-	"github.com/shellhub-io/shellhub/pkg/authorizer"
 	"github.com/shellhub-io/shellhub/pkg/models"
 )
 
-var ErrForbidden = errors.New("resource forbidden")
+// InvalidRole is the number returned when a role is not found in Roles.
+const InvalidRole = -1
+
+const (
+	RoleObserver      = "observer"
+	RoleOperator      = "operator"
+	RoleAdministrator = "administrator"
+	RoleOwner         = "owner"
+)
+
+const (
+	// RoleObserverCode is a role code for observer.
```

Comment:
Why the invalid isn't defined here with the other codes?

Reply:
https://github.com/shellhub-io/shellhub/pull/1834/commits/4ccf4a4e64175d6b9189909d289ccd862652be9b#diff-471917d73591cfd1c57419b9cc8ed82462afe06ae236dc6587c899758fcb8097R15

I've added the code out of iota because of its value: `-1`.